### DISCUSSION TITLE
Automatic Synchronization in QField After Offline Data Collection.

### DISCUSSION THREAD

**Discussion by mahmoodi-source** [opengisch/QField#5762] [Q&A]
Hi,
I'm curious if QField supports automatic synchronization for data collected offline. Specifically, the ability for data collected in offline mode, in an area without internet access, to be automatically uploaded or synced once the device reconnects to WiFi or the internet, without the need to manually push the update. This would greatly improve the workflow for users working in areas with intermittent connectivity.
Thanks for any insights!

**Comment by m-kuhn** [comment]
This is supported via auto-push, you might find this documentation interesting https://docs.qfield.org/get-started/tutorials/advanced-setup-qfc/#enabling-automatic-pushing-of-changes-to-qfieldcloud
